src/components/common/Tabs/TabItem.tsx

```diff
@@ -25,8 +25,13 @@ const Wrapper = styled.li<WrapperProps>`
   font-size: var(--font-size-large);
   letter-spacing: 0.1rem;
   text-align: center;
+  appearance: none;
+  border: 0;
   justify-content: center;
   cursor: pointer;
+  outline: 0;
+
+  /* TODO: Provide alternative :focus styling because of using outline: 0; */
```

Comment:
is this sth u intended to do in this PR and forgot, or sth we need to address in the future?
not sure if I understand

Reply:
Intended to be addressed in the future. As this would apply to a lot more items.

Reply:
Okay. Maybe here is not worth it. For bigger things, or things we don't want to forget, we add a task as an issue or in the backlog so we don't forget. Although now we are not attending so much issues of this gpv1